setup/kubernetes/openshift.md

```diff
@@ -175,3 +175,14 @@ spec:
 When creating workspaces,
 [configure Coder to connect to the internal OpenShift registry](../../admin/registries/index.md)
 and use the base image you just created.
+
+## Next steps
+
+If you have already installed Coder or are using our hosted beta, you can add
```

Comment:
I'd skip the hosted WSP stuff for OpenShift. We are focused on cloud managed Kubernetes deployments for now.